Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/Tools/DCC/Blender/settings.json

```diff
@@ -1,16 +1,16 @@
 {
-	"O3DE_DCCSI_TOOLS_DCC_BLENDER":"True",
-	"DCCSI_BLENDER_VERSION":"3.1",
-	"DCCSI_BLENDER_LOCATION":"C:\Program Files\Blender Foundation\Blender 3.1",
-	"DCCSI_BLENDER_EXE":"C:\Program Files\Blender Foundation\Blender 3.1\blender.exe",
-	"DCCSI_BLENDER_LAUNCHER":"C:\Program Files\Blender Foundation\Blender 3.1\blender-launcher.exe",
-	"DCCSI_BLENDER_PYTHON":"C:\Program Files\Blender Foundation\Blender 3.1\3.1\python",
-	"DCCSI_BLENDER_PY_BIN":"C:\Program Files\Blender Foundation\Blender 3.1\3.1\python\bin",
-	"DCCSI_BLENDER_PY_EXE":"C:\Program Files\Blender Foundation\Blender 3.1\3.1\python\bin\python.exe",
-	"DCCSI_BLENDER_SET_CALLBACKS":"True",
-	"DCCSI_BLENDER_SCRIPTS":"C:\Program Files\Blender Foundation\Blender 3.1\3.1\scripts",
-	"DCCSI_BLENDER_ADDONS":"C:\Program Files\Blender Foundation\Blender 3.1\3.1\scripts\addons",
-	"DCCSI_PY_VERSION_MAJOR":"3",
-	"DCCSI_PY_VERSION_MINOR":"10",
-	"DCCSI_PY_VERSION_RELEASE":"2"
+	"O3DE_DCCSI_TOOLS_DCC_BLENDER": "True",
+	"DCCSI_BLENDER_VERSION": "3.1",
+	"DCCSI_BLENDER_EXE": "C:/Program Files/Blender Foundation/Blender 3.1/blender.exe",
```

Comment:
I'm assuming this is the default install location for blender. There isn't another way to find this PATH dynamically is there?

Reply:
These are the defaults for the most current version.

This is something I am working through incrementally. Yes, in a future iteration will implement something like discovery.py
This module could use a combination of known default location and/or locate via registry, and this could be called the first run when setting up the DCC app.

There is definitely going to be a common edge case of multiple version installs; and teams may want to be specific about the version used (they may want to lock it for some time.)  So discovery.py would be a good opportunity to let the user know multiple are discovered and prompt the user to decide which to use.

config.py is actually dynamic (env and settings logic) an layered, for instance can load settings.local.json to override anything via the settings file (this is so a dev can locally make any setting change.)

But you make a good point (I think) that probably most of these values shouldn't need to be set in this file.
I think I put them here, so that someone poking around would immediately know what settings they could modify and override without going into the code.  I think what might be best then, is to generate a settings.json.example - this would have all that example info but not be loaded.

Reply:
@cgalvan is this sufficient for this first prototype? I will do discovery and put that plumbing in place next PR. If so, please resolve this comment.